docs/Developers/Review.rst

```diff
@@ -65,6 +65,8 @@ The review process for coala is as follows:
    ``corobo mark wip <URL>`` or ``corobo mark pending <URL>``.
 3. The developers will acknowledge the commits by writing
 
+   * In case an owner is reviewing it:
```

Comment:
Better to use `maintainer` instead of `owner`.
Otherwise LGTM 😄 
cc @Makman2

Reply:
But I think it is mentioned in the contributors page -- owner/member

Reply:
Done @sangamcse  🎉